bonus_guides/G_static_assets.md

```diff
@@ -1 +1,289 @@
-### FPO
+Instead of implementing its own asset pipeline Phoenix uses [Brunch](http://brunch.io), a fast and developer-friendly asset build tool. Phoenix comes with a default configuration for Brunch and it will work out of the box, but it is very easy to bend it to our needs, add support for various script and style languages, like CoffeeScript, JSX, or LESS.
+
+Brunch has [a very good tutorial](https://github.com/brunch/brunch-guide), but this short guide should be enough to get us started with asset management from the Phoenix perspective.
+
+
+#### Installation
+
+Brunch is a [Node.js](https://nodejs.org/) application. A newly generated Phoenix project contains `package.json` which lists packages for installation with [npm](https://www.npmjs.com/), the Node Package Manager . If we agree to install dependencies when running `mix phoenix.new`, Phoenix will run `npm` for us. If we don't, or if we change `package.json`, we can always do this ourselves:
+
+```
+npm install
+```
+
+This will install Brunch and its plugins into directory `node_modules`.
+
+
+#### Default Configuration And Workflow
+
+The second important file is `brunch-config.js`. This is a configuration for Brunch itself. Let's see how it configures asset management for Phoenix.
```

Comment:
Also, perhaps we could show the contents of the file that demonstrate the next sentence?

```javascript
. . . 
conventions: {
    // This option sets where we should place non-css and non-js assets in.
    // By default, we set this to '/web/static/assets'. Files in this directory
    // will be copied to `paths.public`, which is "priv/static" by default.
    assets: /^(web\/static\/assets)/
  },
. . .
```